api/pkg/s3/bucketacl.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package s3
+
+import (
+	"encoding/xml"
+	"io"
+	"io/ioutil"
+	"strings"
+
+	"github.com/emicklei/go-restful"
+	"github.com/journeymidnight/yig/helper"
+	"github.com/opensds/multi-cloud/api/pkg/common"
+	"github.com/opensds/multi-cloud/api/pkg/s3/datatype"
+	s3error "github.com/opensds/multi-cloud/s3/error"
+	pb "github.com/opensds/multi-cloud/s3/proto"
+	log "github.com/sirupsen/logrus"
+)
+
+func (s *APIService) BucketAclPut(request *restful.Request, response *restful.Response) {
+	bucketName := strings.ToLower(request.PathParameter(common.REQUEST_PATH_BUCKET_NAME))
+	log.Infof("received request: PUT bucket[name=%s] acl\n", bucketName)
+
+	var err error
+	var acl datatype.Acl
+	var policy datatype.AccessControlPolicy
+	if _, ok := request.Request.Header[common.REQUEST_HEADER_ACL]; ok {
+		acl, err = getAclFromHeader(request)
+		if err != nil {
+			WriteErrorResponse(response, request, err)
+			return
+		}
+	} else {
```

Comment:
Just run"Go Import file" to update indentation in your current file

Reply:
Thanks for reviewing,  you mean format the file?  I use gofmt  to format the file , and nothing happened for the file